cmake/templates/cmake_variables.rst.in

```diff
@@ -31,43 +31,43 @@ The options are split into these categories:
 Additional tools and libraries used by |hpx|
 --------------------------------------------
 
-Here is a list of additional libraries and tools which are either optionally
+Here is a list of additional libraries and tools that are either optionally
 supported by the build system or are optionally required for certain examples or
 tests. These libraries and tools can be detected by the |hpx| build system.
 
 Each of the tools or libraries listed here will be automatically detected if
 they are installed in some standard location. If a tool or library is installed
-in a different location you can specify its base directory by appending
+in a different location, you can specify its base directory by appending
 ``_ROOT`` to the variable name as listed below. For instance, to configure a
 custom directory for ``BOOST``, specify ``BOOST_ROOT=/custom/boost/root``.
 
 .. option:: BOOST_ROOT:PATH
 
-   Specifies where to look for the |boost|_ installation to be used for
-   compiling |hpx| Set this if CMake is not able to locate a suitable version of
-   |boost|_ The directory specified here can be either the root of a installed
-   Boost distribution or the directory where you unpacked and built |boost|_
+   Specifies where to look for the Boost installation to be used for
+   compiling |hpx|. Set this if CMake is not able to locate a suitable version of
+   Boost. The directory specified here can be either the root of an installed
+   Boost distribution or the directory where you unpacked and built Boost
```

Comment:
I just want to check that the `|boost|_` and `|cmake|_` to `Boost` and `CMake` replacements are intentional. `|boost|` will be rendered as `Boost`, and `|boost|_` will be rendered as as a clickable link with the text `Boost`.

Reply:
It's intentional. The use of hyperlinks is inconsistent throughout the manual, so the others and I decided to only include external links the first time a term is mentioned, and to include internal links within the manual itself every time a term is mentioned.

Reply:
Sounds good, thanks!